ai/clickhouse.go

```diff
@@ -0,0 +1,117 @@
+package ai
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"math"
+	"strings"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+	"github.com/Masterminds/squirrel"
+)
+
+const maxClickhouseResultRows = 1000
+
+type AIStreamStatusEventRow struct {
+	StreamID      string   `ch:"stream_id"`
+	AvgInputFPS   float64  `ch:"avg_input_fps"`
+	AvgOutputFPS  float64  `ch:"avg_output_fps"`
+	ErrorCount    uint64   `ch:"error_count"`
+	Errors        []string `ch:"errors"`
+	TotalRestarts uint64   `ch:"total_restarts"`
+	RestartLogs   []string `ch:"restart_logs"`
```

Comment:
We should probably have corresponding `json` flags here for proper JSON serialization of the response? We're using `snake_case` on the AI APIs, at least.

Or doest the `ch` tag also set the JSON field name?